homeassistant/components/roomba/__init__.py

```diff
@@ -92,6 +92,7 @@
         raise exceptions.ConfigEntryNotReady
 
     hass.data[DOMAIN][config_entry.entry_id] = roomba
+    hass.data[DOMAIN][CONF_BLID] = config_entry.data[CONF_BLID]
```

Comment:
```suggestion
    hass.data[DOMAIN][CONF_BLID] = config_entry.data[CONF_BLID]
```

Since you can have more than one this will conflict.  You can store a dict instead.

```
 hass.data[DOMAIN][config_entry.entry_id] = {
   ROOMBA: roomba,
  BLID: config_entry.data[CONF_BLID]
 }

```